src/main/java/se/poochoo/reminder/ReminderHelper.java

```diff
@@ -3,29 +3,32 @@
 import android.app.Notification;
 import android.app.NotificationManager;
 import android.app.PendingIntent;
-import android.content.BroadcastReceiver;
 import android.content.Context;
 import android.content.Intent;
 import android.media.RingtoneManager;
 import android.net.Uri;
-import android.provider.ContactsContract;
-
-import com.google.android.gms.internal.di;
+import android.util.SparseIntArray;
 
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Timer;
 
 import se.poochoo.DialogActivity;
 import se.poochoo.R;
+import se.poochoo.RowItem;
 import se.poochoo.cardsui.CardTicker;
 import se.poochoo.proto.Messages;
 
-/**
- * Created by Erik on 2013-12-08.
- */
 public class ReminderHelper {
 
+    static final long UPDATE_NOTIFICATION_INTERVAL = 30000;
```

Comment:
The variables already affected by this pull request have been fixed. The ones that are not were not as that would require including more files into this PR